fakeable_external_test.go

```diff
@@ -0,0 +1,660 @@
+package gofakeit_test
+
+import (
+	"math/rand"
+	"testing"
+	"time"
+
+	"github.com/brianvoe/gofakeit/v6"
+)
+
+var (
+	testTimeValue = time.Now()
+)
+
+type CustomString string
+
+func (c CustomString) Fake(faker *gofakeit.Faker) interface{} {
+	return CustomString("hello test")
+}
+
+type CustomBool bool
+
+func (c CustomBool) Fake(faker *gofakeit.Faker) interface{} {
+	return CustomBool(true)
+}
+
+type CustomInt int
+
+func (c CustomInt) Fake(faker *gofakeit.Faker) interface{} {
```

Comment:
Can you provide a couple of different ints, uints and floats to compare them a little more to a representation of what various ints, uints and floats would be?